src/alt-gosling-model/alt-structure/chart-types.ts

```diff
@@ -56,8 +56,8 @@ export function determineSpecialCases(altTrack: AltTrackSingle | AltTrackOverlai
     if (_mark === 'rule' && _allEncodings.includes('y')) {
         return `${layoutDesc}chart with horizontal lines`;
     }
-    if (markToText.get(_mark)) {
-        return `${layoutDesc}chart with ${markToText.get(_mark)}`;
+    if (callMarkToText(_mark)) {
+        return `${layoutDesc}chart with ${callMarkToText(_mark)}`;
     }
     
     return `unknown chart`;
```

Comment:
Do we still need this? I think `callMarkToText` always returns something, so this line of code seems to be unreachable. In that case, we can remove this line and also remove the if statement right above:

```diff
- if (callMarkToText(_mark)) {
    return `${layoutDesc}chart with ${callMarkToText(_mark)}`;
- }
```

Reply:
Although these seem equivalent in information